### PR TITLE
{bio}[intel/2017b] BWA v0.7.17

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2500.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/45761908223d7be57fa68b5af350b945 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2047.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/64a75e68db4dba23e39f72a2277308c6 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
rescomp1 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.14
See https://gist.github.com/bd7c857b1025041934364a5d87b77f30 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
rescomp1 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.14
See https://gist.github.com/de62d30d8f4cc3e3973a0d8327f7ef48 for a full test report.

File: easybuild/easyconfigs/b/BWA/BWA-0.7.17-intel-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+#
+# Version >= 0.7.15
+# Author: Adam Huffman
+# The Francis Crick Institute
```
Comment (verdurin) [inline easybuild/easyconfigs/b/BWA/BWA-0.7.17-intel-2017b.eb]: @boegel - could you change my affiliation here? Should be:
Big Data Institute, University of Oxford

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**